domains/srm.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "srmlab002",
+        "email": "8zaqibue@duck.com"
+    },
+    "record": {
+        "MX": ["srmt3.uk.to"]
+    }
+}
```

Comment:
is this all you need for your email service? i think you also need to add txt verification here